src/mood.ts

```diff
@@ -126,6 +128,19 @@ export function meatMood(urKels = false, embezzlers = false): Mood {
     }
   }
 
+  const canRecord =
+    getWorkshed() === $item`warbear LP-ROM burner` ||
+    have($item`warbear LP-ROM burner` || get("questG04Nemesis") === "finished");
+
+  if (myClass() === $class`Accordion Thief` && myLevel() >= 15 && !canRecord) {
```

Comment:
I feel like this is gonna hit the song limit all the time, if it doesn't have priority over the other songs.

Reply:
The way our song slots are set up should make that impossible